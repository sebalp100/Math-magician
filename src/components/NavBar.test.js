import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Nav from './NavBar';

describe('Test header component', () => {
  it('header renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Nav />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
