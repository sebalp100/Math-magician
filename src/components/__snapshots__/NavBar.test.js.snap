// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test header component header renders correctly 1`] = `
<nav>
  <h1>
    Math Magicians
  </h1>
  <div
    className="link"
  >
    <li>
      <a
        href="/"
        onClick={[Function]}
        style={
          Object {
            "color": "azure",
            "textDecoration": "none",
          }
        }
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/calculator"
        onClick={[Function]}
        style={
          Object {
            "color": "azure",
            "textDecoration": "none",
          }
        }
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/quote"
        onClick={[Function]}
        style={
          Object {
            "color": "azure",
            "textDecoration": "none",
          }
        }
      >
        Quote
      </a>
    </li>
  </div>
</nav>
`;
